composer.json

```diff
@@ -120,6 +120,7 @@
         "symfony/yaml": "self.version"
     },
     "require-dev": {
+        "ext-openssl": "*",
```

Comment:
this should be removed. If openssl is needed to run some tests, we should skip those tests when the extension is missing instead of forbidding to install the dev setup of Symfony entirely (maybe the contributor wants to work on a totally different part of the system)